vllm/config.py

```diff
@@ -772,15 +770,10 @@ def __init__(self,
                 # It is the values that have the best balance between ITL
                 # and TTFT on A100. Note it is not optimized for throughput.
                 self.max_num_batched_tokens = 512
-            elif model_mode == ModelMode.EMBEDDING:
-                # For embedding, choose specific value for higher throughput
+            elif model_mode in [ModelMode.EMBEDDING, ModelMode.SIMPLE]:
                 self.max_num_batched_tokens = max(
-                    max_model_len, _EMBEDDING_MODEL_MAX_NUM_BATCHED_TOKENS)
-            elif model_mode == ModelMode.SIMPLE:
-                # For non-[decoder,embedding] model, choose specific value
-                # for higher throughput
-                self.max_num_batched_tokens = max(
-                    max_model_len, _SIMPLE_MODEL_MAX_NUM_BATCHED_TOKENS)
+                    max_model_len,
+                    ModelMode.get_model_max_num_batched_tokens(model_mode))
```

Comment:
How about we return `None` for the other model modes (so we don't need the `elif` statement for this)? Afterwards we only apply `max` if the model mode returns a valid integer.

Reply:
Only keep "if enable_chunked_prefill" "else"?   remove "elif:"?

Reply:
Yeah. We can apply the `max` part inside the `else` block

Reply:
Sorry, didn't get your point.